pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -101,7 +103,13 @@ func (n *NodeClaim) Add(pod *v1.Pod) error {
 
 	// Check instance type combinations
 	requests := resources.Merge(n.Spec.Resources.Requests, resources.RequestsForPods(pod))
+
 	filtered := filterInstanceTypesByRequirements(n.InstanceTypeOptions, nodeClaimRequirements, requests)
+
+	if len(filtered.requirementInCompatibleWithMinValues) > 0 {
```

Comment:
```suggestion
	if len(filtered.requirementIncompatibleWithMinValues) > 0 {
```

nit: Incompatible is one word so we shouldn't camel case inside of it

Reply:
Addressed.